driver/src/main/scala/org/mongodb/scala/ChangeStreamObservable.scala

```diff
@@ -140,7 +139,15 @@ case class ChangeStreamObservable[TResult](private val wrapped: ChangeStreamIter
    * @tparam T the result type
    * @return an Observable
    */
-  def withDocumentClass[T](clazz: Class[T]): Observable[T] = observe(wrapped.withDocumentClass(clazz))
+  def withDocumentClass[T](clazz: Class[T]): Observable[T] = wrapped.withDocumentClass(clazz).toObservable()
```

Comment:
Had to help the compiler here, so `Publisher[T]` can be implicitly converted to `Observable[T]`. 

This is the only place I've had to explicitly use the `toObservable()` helper. That in itself is odd as the `toObservable` method is an implicit method on the implicit conversion class that converts `Publisher[T]` to `Observable[T]`.

Reply:
I think this is more of a Scala implicit resolution issue - I tried to work around it but failed to find the solution in this case.

Reply:
ack